lib/utilities/get-config.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+let Yam = require('yam');
+let Project = require('../models/project');
+let config = new Yam('ember-cli', {
+  primary: Project.getProjectRoot(),
+});
+
+module.exports = function getConfig(mockedConfig) {
```

Comment:
allow passing in a mocked config to support current behavior in some tests, e.g. `options.Yam` in `cli/index-test.js`